src/wireviz/Harness.py

```diff
@@ -296,7 +296,11 @@ def output(self, filename: (str, Path), view: bool = False, cleanup: bool = True
         # HTML output
         with open_file_write(f'{filename}.html') as file:
             file.write('<!DOCTYPE html>\n')
-            file.write('<html><head><meta charset="UTF-8"></head><body style="font-family:Arial">')
+            file.write('<html lang="en"><head>\n')
+            file.write(' <meta charset="UTF-8">\n')
+            file.write(f' <meta name="generator" content="wireviz {__version__}" />\n')
```

Comment:
Should we also add the Wireviz URL? How?

Reply:
I am not too familiar with these `<meta>` tags... perhaps a HTML comment after this line is enough for the URL?

Reply:
I didn't find any suitable [`<link>` relation values](https://www.w3schools.com/tags/att_link_rel.asp), [`<meta>` name values](https://www.w3schools.com/tags/att_meta_name.asp), [`<meta>` attributes](https://www.w3schools.com/tags/tag_meta.asp) or [global attributes](https://www.w3schools.com/tags/ref_standardattributes.asp), and was thinking just appending the URL after the version number, but I've never seen that used by others, so your suggestion is probably better.

Is it OK to keep the name as `WireViz` in the `.gv` output and `wireviz` most other places, or do you consider it unfortunate?

Reply:
If it's meant to be human-readable (such as here), `WireViz` looks better IMHO. `wireviz` should be reserved for CLI commands etc.

[This site](http://www.meta-tags.de/meta_name_generator) even includes a copyright notice inside the `<meta>` tag... so it wouldn't really matter if the URL was in there too, after the version number.

`<meta name="generator" content="WireViz {__version__} - https://github.com/formatc1702/WireViz/">`

Reply:
I edited my previous comment with a new suggestion before I discovered your last comment, but perhaps your last suggestion is a cleaner solution?

Reply:
I think one line to include version and URL should be enough :) Thanks for looking into this.